pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -75,6 +75,13 @@ type JobSpec struct {
 
 	// Required when the action is `export`
 	Export *ExportOptions `json:"export,omitempty"`
+	// Required when the action is `sync`
+	Sync *SyncOptions `json:"sync,omitempty"`
+}
+
+type SyncOptions struct {
+	// Force forces the sync to overwrite
+	Force bool `json:"force,omitempty"`
```

Comment:
Is "force" the right word?  I can think of three "sync" options -- force is the one I am not sure we need:

1. **Incremental** (if if the repo supports it.... eg git).   
       *  walk though all the CREATE/UPDATE/DELETE commands since the last hash we saved

2. **Complete** / Full / Normal?  not sure what to call it :)  this is what file system will always do
       * list everything in grafana
       * list everything in the repo
       * compare the differences (grafana returns hash!)
       * apply any necessary changes

3. **Force**  (maybe not necessary).   
       * Delete everything from grafana
       * List everything from the repository
       * Write everything to grafana

Reply:
Ok, turned it into a `complete` mode and put the `force` aside for now.